packages/app/src/utils/tooltips.ts

```diff
@@ -0,0 +1,46 @@
+export const tooltips = {
+  yourStake: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy 
+  eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. 
+  At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, 
+  no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  totalStatke: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, 
+  sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam 
+  erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea 
+  rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  offsetCO2: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam 
+  nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam 
+  voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita 
+  kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  withdraw: `Delayed withdrawal is always free. You will receive a ticket and at the
```

Comment:
```suggestion
  withdraw: `Withdraw your funds by converting gSOL back to SOL. You can choose between immediate and delayed withdrawal. Immediate withdrawal draws funds from the Marinade liquidity pool. If the pool is low, it may incur a fee. Delayed withdrawal is always free. You will receive a ticket which can be redeemed at the end of the Solana Epoch to receive your SOL.
```

This might be too long. If so, then replace most of it with a link to the docs.